xml/System.Reflection.Metadata/ManifestResource.xml

```diff
@@ -102,8 +102,8 @@
         <ReturnType>System.Reflection.Metadata.EntityHandle</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary><see cref="T:System.Reflection.Metadata.AssemblyFileHandle" />, <see cref="T:System.Reflection.Metadata.AssemblyReferenceHandle" />, or a handle where the <see cref="P:System.Reflection.Metadata.EntityHandle.IsNil" /> property is <see langword="true" />.</summary>
+        <value>An <see cref="T:System.Reflection.Metadata.EntityHandle" /> instance.</value>
```

Comment:
```suggestion
        <value>Gets an <see cref="T:System.Reflection.Metadata.AssemblyFileHandle" /> or an <see cref="T:System.Reflection.Metadata.AssemblyReferenceHandle" />. If the <see cref="P:System.Reflection.Metadata.EntityHandle.IsNil" /> property is <see langword="true" />, it returns a handle with default values.</value>
```

Reply:
I moved the summary to the value. @mairaw does this description look better?

Reply:
Yes, your suggestion looks good. Don't you want to add the summary too?

Reply:
I'm not sure what the summary for this would be.
@tmat and @nguerrera, can you jump in to help with some of the clarifications in the comments?